lib/engine/game/g_1870/step/check_connection.rb

```diff
@@ -53,13 +53,17 @@ def actions(_entity)
             ACTIONS
           end
 
+          # This step should not be passed
```

Comment:
it'd be better to override operating.rb to change the behaviour to skip force next entity on check_connection.

Reply:
That requires copy-pasting the entire function, which is very likely that someone will want to change in the future